tests/test_validators.py

```diff
@@ -7,10 +7,19 @@
 
 @pytest.mark.parametrize('key', SCHEMAS.keys())
 def test_validation(key):
-    validate(key, EXEMPLARS[key], SCHEMAS)
+    if isinstance(EXEMPLARS[key], list):
```

Comment:
Testing against Python lists, rather than `collections.abc.Sequence` and not str